homeassistant/components/nut/sensor.py

```diff
@@ -85,15 +85,15 @@ def __init__(
         self._attr_device_info.update(data.device_info)
 
     @property
-    def native_value(self):
+    def native_value(self) -> str | None:
         """Return entity state from ups."""
-        status = self.coordinator.data
+        status: dict[str, str] = self.coordinator.data
```

Comment:
You should be able to avoid this if you add the type to the Coordinator https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/update_coordinator.py#L36

Reply:
Example: https://github.com/bdraco/home-assistant/commit/c5a9c48db79a85b800164992396c21d69ea45cc1